app/models/qdm/tacoma/individual_result.rb

```diff
@@ -0,0 +1,33 @@
+module QDM
+  # IndividualResult stores the patient-level (population/clause) results for a patient/measure combination
+  class IndividualResult
+    include Mongoid::Document
+    include Mongoid::Timestamps
+
+    # Population Attributes
+    field :STRAT, type: Integer
+    field :IPP, type: Integer
+    field :DENOM, type: Integer
+    field :NUMER, type: Integer
+    field :NUMEX, type: Integer
+    field :DENEX, type: Integer
+    field :DENEXCEP, type: Integer
+    field :MSRPOPL, type: Integer
+    field :OBSERV, type: Float
+    field :MSRPOPLEX, type: Integer
+
+    # Result Attributes
+    field :clause_results, type: Hash
+    field :episode_results, type: Hash
+    filed :population_relevance, type: Hash
+    field :statement_relevance, type: Hash
+    field :statement_results, type: String
```

Comment:
In the Javascript version this is a Mixed, but here it's a String.

Reply:
Good catch. This should be a `Mixed`/`Hash`.